CMakeLists.txt

```diff
@@ -178,6 +180,10 @@ endif()
 
 if(VELOX_ENABLE_PARQUET)
   add_definitions(-DVELOX_ENABLE_PARQUET)
+  if(${CMAKE_SYSTEM_NAME} MATCHES "Linux" AND (${CMAKE_SYSTEM_PROCESSOR} MATCHES
```

Comment:
Maybe we should not turn it on by automatically and just leave it to user to turn it on.  We should make sure to turn it on in one of the CircleCI (probably `linux-adapters`) though